test/assets_test.dart

```diff
@@ -8,7 +8,7 @@ import 'dart:async';
 
 import 'package:barback/barback.dart';
 import 'package:code_transformers/assets.dart';
-import 'package:code_transformers/tests.dart';
+import 'package:transformer_test/transformer_test.dart';
```

Comment:
many tools would consider transformer_test.dart as a test file. Maybe rename it to something else? Some ideas:
 * `transformer_testing/transformer_testing.dart`
 * `transformer_test/transformer_test_utils.dart`
 * `transformer_test/utils.dart`
 * `transformer_test/test_harness.dart`